MAUI/Scheduler/getting-started.md

```diff
@@ -89,14 +76,48 @@ namespace GettingStarted
 {% endhighlight %}
 {% endtabs %}
 
+## Step 4: Add a Basic Scheduler view.
```

Comment:
remove dot in heading 
Add a Basic Scheduler view.

Reply:
i have address the correction in the following pr
https://github.com/syncfusion-content/maui-docs/pull/2670/files